automation/ci/run_aio_for_tf.sh

```
@@ -0,0 +1,60 @@
+# run_aio_integration.sh this builds nothing from source -- it pulls a published
+# image and runs the same consul+vault+traefik+DB+AIO compose stack on the runner.
+#
+# Run from the repo root. Env:
+#   JANS_FQDN          required, e.g. tf-123.jans.test (must resolve to 127.0.0.1)
+#   JANS_PERSISTENCE   required, MYSQL | PGSQL
+#   AIO_IMAGE_TAG      optional, default ghcr nightly all-in-one
+
+set -uo pipefail
+
+REPO_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd)"
+cd "$REPO_ROOT"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)